### PR TITLE
115 display image of EAK in "Ausweisen"-Tab

### LINKED ISSUE #115: Display card and card details
View that displays a image of an EAK, name of the EAK owner, expiration date and Kommune. 
If not yet authorized: Link to QR scanner

User story #15

### PR DESCRIPTION
**Pull request by PaulaWi**
closes #115

### CODE REVIEW

**Review by maxammann — APPROVED** [review]

File: frontend/pubspec.yaml [hunk 1]
```diff
@@ -61,6 +63,7 @@ flutter:
   uses-material-design: true
 
   assets:
+    - assets/
     - assets/category_icons/
     - assets/detail_headers/
```
Comment (elKei24) [inline frontend/pubspec.yaml:68]: We probably do not need the subfolders any more now, right?

File: frontend/lib/identification/scanner/qr_code_scanner.dart [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    });
+  }
+
+  void _tryParseCodeContent(String codeContent) {
+    print("Scan successful, reading payload...");
+    try {
+      final cardDetails = parseQRCodeContent(codeContent);
+      // TODO this method gets called multiple times in a row for some reason
+      // The following code should be exclusive, but dart does not natively
+      // support mutexes, synchronized, etc.
```
Comment (elKei24) [inline frontend/lib/identification/scanner/qr_code_scanner.dart:188]: Is this TODO still there on purpose?

File: frontend/lib/identification/scanner/qr_code_scanner.dart [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+                    ],
+                  ),
+                ],
+              ),
+            ),
+          )
+        ],
+      ),
+    );
+  }
```
Comment (elKei24) [inline frontend/lib/identification/scanner/qr_code_scanner.dart:128]: This is pretty big, we could think about extracting parts of that into a separate build function/widget.

File: frontend/lib/identification/scanner/qr_code_scanner.dart [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+    this.controller = controller;
+    controller.scannedDataStream.listen((scanData) {
+      if (scanData != null) {
+        _tryParseCodeContent(scanData.code);
+      }
+    });
+  }
+
+  void _tryParseCodeContent(String codeContent) {
+    print("Scan successful, reading payload...");
```
Comment (elKei24) [inline frontend/lib/identification/scanner/qr_code_scanner.dart:183]:
```suggestion
```

**Review by elKei24 — APPROVED** [review]
Comments mainly regard QR code stuff

File: frontend/pubspec.yaml (hunk 1 above)
Comment (PaulaWi) [reply frontend/pubspec.yaml:68]: Subdirectories are not included automatically so we have to create an entry for every subdirectory

Comment (elKei24) [reply frontend/pubspec.yaml:68]: Oh, okay, good to know 👍🏽

File: frontend/lib/identification/scanner/qr_code_scanner.dart (hunk 2 above)
Comment (RoAnBu) [reply frontend/lib/identification/scanner/qr_code_scanner.dart:188]: Yes

**Review by maxammann — APPROVED** [review]

File: frontend/lib/identification/scanner/qr_code_scanner.dart (hunk 3 above)
Comment (RoAnBu) [reply frontend/lib/identification/scanner/qr_code_scanner.dart:128]: To be refactored later when revisited

**Comment by RoAnBu** [comment]
image

**Pull request merged**